libs/pages/clusters/src/lib/feature/page-clusters-create-feature/step-summary-feature/step-summary-feature.tsx

```diff
@@ -84,6 +100,31 @@ export function StepSummaryFeature() {
   }, [pathCreate, generalData, navigate, organizationId])
 
   const onSubmit = async (withDeploy: boolean) => {
+    if (generalData && generalData.installation_type === 'SELF_MANAGED' && kubeconfigData) {
+      try {
+        await createCluster({
+          organizationId,
+          clusterRequest: {
+            name: generalData.name,
+            description: generalData.description,
+            region: generalData.region,
+            cloud_provider: generalData.cloud_provider,
+            kubernetes: 'SELF_MANAGED',
+            production: true,
+            ssh_keys: [],
+            kubeconfig: kubeconfigData.file_content,
+            features: [],
+          },
+        })
+        navigate({
+          pathname: CLUSTERS_URL(organizationId) + CLUSTERS_GENERAL_URL,
+          search: '?showSelfManagedGuide',
```

Comment:
`show-self-managed-guide` should be more elegant for URLs no?

Reply:
https://github.com/Qovery/console/pull/1117/commits/c6819e6037e3f519723dba6707db20c10b6f7028